orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/BatchStepStatus.groovy

```diff
@@ -41,6 +41,7 @@ class BatchStepStatus {
       case ExecutionStatus.TERMINAL:
         return new BatchStepStatus(RepeatStatus.FINISHED, result.status.exitStatus, BatchStatus.FAILED)
       case ExecutionStatus.RUNNING:
+      case ExecutionStatus.PAUSED:
```

Comment:
EchoNotifyingStageExecutionListener:
    if (stepExecution.status.running) {
      return
    }

I wouldn't expect it to emit an event ... we only send events to echo before/after task whereas `PAUSED` occurs during task execution.